README.md

```diff
@@ -20,13 +20,85 @@ cd webapp
 mvn spring-boot:run
 ```
 ### To start REST:
+####Development Profile (runs h2 database)
 ```
 cd rest
-mvn spring-boot:run
+mvn spring-boot:run -Pdev
+```
+####Production Profile (runs mysql database)
+```
+cd rest
+mvn spring-boot:run -Pprod
```

Comment:
This is information to run application in development mode with Maven.
Please provide additional information how run executable jar without Maven